code/modules/spacepods/spacepod.dm

```diff
@@ -54,12 +68,38 @@
 	var/move_delay = 2
 	var/next_move = 0
 
+/obj/spacepod/proc/apply_paint(mob/user as mob)
+	var/part_type
+	var/part = input(user, "Choose part", null) as null|anything in list("Lights","Rim","Paint","Windows")
+	if(part == "Lights")
```

Comment:
why have the list be text strings, then end up matching them to defines? Why not just have the defines in the list?